types/jest/index.d.ts

```diff
@@ -28,6 +28,7 @@
 //                 Regev Brody <https://github.com/regevbr>
 //                 Alexandre Germain <https://github.com/gerkindev>
 //                 Adam Jones <https://github.com/domdomegg>
+//                 Sandino Nuñez <https://github.com/sandinosaso>
```

Comment:
You've added yourself as an owner in this PR. You're welcome to become one - but perhaps review https://github.com/DefinitelyTyped/DefinitelyTyped#user-content-definition-owners to understand what the implications of this are. Maybe consider becoming an owner if you find yourself contributing to the Jest types multiple times, rather than on this first PR?

Reply:
Yes, I did not read about it before adding me. It is my first contribution I will remove myself if in the future there are more maybe I can think about it. Thank you for noticing that :)